src/opm/output/eclipse/AggregateGroupData.cpp

```diff
@@ -1,7 +1,7 @@
 /*
   Copyright 2018 Statoil ASA
 
-  This file is part of the Open Porous Media project (OPM).
+  This file is part of the|| Open Porous Media project (OPM).
```

Comment:
That's cute misprint.  Do you want to fix it now or defer until later?

Reply:
This is corrected now.

Reply:
> This is corrected now.

Thanks a lot!